File: fibs/js/fibs.js
/* Use like so:
 *     node fibs.js [n]
 * where n is the nth Fibonnaci number you wish to calculate. */

/* Generates the nth Fibonacci number. */
function fibs(n) {
  if (n <= 1) {
    return 1;
  } else {
    return fibs(n - 1) + fibs(n - 2);
  }
}

console.log(fibs(parseInt(process.argv[2])));

File: fibs/java/Fibs.java
public class Fibs {

    static int fibs(int n) {
        if (n <= 1) {
            return 1;
        } else {
            return fibs(n - 1) + fibs(n - 2);
        }
    }

    public static void main(String[] args) {
        System.out.println(fibs(Integer.parseInt(args[0])));
    }

}
